web/src/components/tasks/anytime-action/tax-clearance-certificate/steps/Requirements.tsx

```diff
@@ -1,17 +1,35 @@
+import { NeedsAccountModal } from "@/components/auth/NeedsAccountModal";
 import { Content } from "@/components/Content";
 import { HorizontalLine } from "@/components/HorizontalLine";
 import { CtaContainer } from "@/components/njwds-extended/cta/CtaContainer";
 import { LiveChatHelpButton } from "@/components/njwds-extended/LiveChatHelpButton";
 import { PrimaryButton } from "@/components/njwds-extended/PrimaryButton";
 import { ActionBarLayout } from "@/components/njwds-layout/ActionBarLayout";
+import { AuthContext } from "@/contexts/authContext";
+import { NeedsAccountContext } from "@/contexts/needsAccountContext";
+import { IsAuthenticated } from "@/lib/auth/AuthContext";
 import { useConfig } from "@/lib/data-hooks/useConfig";
-import { ReactElement } from "react";
+import { ROUTES } from "@/lib/domain-logic/routes";
+import { ReactElement, useContext } from "react";
 
 interface Props {
   setStepIndex: (step: number) => void;
 }
+
 export const Requirements = (props: Props): ReactElement => {
   const { Config } = useConfig();
+  const { state: authState } = useContext(AuthContext);
+  const { setShowNeedsAccountModal } = useContext(NeedsAccountContext);
```

Comment:
pebble - NeedsAccountContext can return isAuthenticated, AuthContext is not needed

Reply:
BusinessFormationPaginator.tsx is a great example to review

Reply:
Thanks for this pointer!!